tools/harness-simulation/posix/launch_testbed.py

```diff
@@ -155,15 +174,21 @@ def main():
     args = parser.parse_args()
 
     # Check validation of arguments
-    if not 0 <= args.ot11_num <= MAX_OT11_NUM:
-        raise ValueError(f'The number of FTDs should be between 0 and {MAX_OT11_NUM}')
+    if args.ot_num < 0:
+        raise ValueError(f'The number of FTDs should be non-negative')
+
+    if args.otbr_num < 0:
+        raise ValueError(f'The number of OTBRs should be non-negative')
 
     if not 0 <= args.sniffer_num <= MAX_SNIFFER_NUM:
-        raise ValueError(f'The number of FTDs should be between 0 and {MAX_SNIFFER_NUM}')
+        raise ValueError(f'The number of sniffers should be between 0 and {MAX_SNIFFER_NUM}')
 
-    if args.ot11_num == args.sniffer_num == 0:
+    if args.ot_num == args.otbr_num == args.sniffer_num == 0:
         raise ValueError('At least one device is required')
```

Comment:
I guess it's also an invalid case if there's only a sniffer.

Reply:
Yes. I will change the logic here.